packages/liveblocks-react-ui/src/components/internal/AiChatUserMessage.tsx

```diff
@@ -19,14 +20,14 @@ export interface AiChatUserMessageProps extends ComponentProps<"div"> {
   overrides?: Partial<GlobalOverrides>;
 }
 
+function PlainTextPart({ part }: { part: AiTextPart }) {
```

Comment:
Not now but when it will become public, each part's props type should be exported to avoid `{ part: AiTextPart }` in user code